bundles/org.eclipse.ui.ide/src/org/eclipse/ui/internal/ide/dialogs/OpenResourceQuickAccessComputer.java

```diff
@@ -107,7 +107,8 @@ public String getLabel() {
 
 		@Override
 		public ImageDescriptor getImageDescriptor() {
-			return ImageDescriptor.createFromImageDataProvider(zoom -> fLabelProvider.getImage(fFile).getImageData());
+			return ImageDescriptor
+					.createFromImageDataProvider(zoom -> fLabelProvider.getImage(fFile).getImageData(zoom));
```

Comment:
If you like this particular style, you can try `fLabelProvider.getImage(file)::getImageData` (so the zoom isn't even to be mentioned)

Reply:
Thank you! Makes sense. Didn't think about that when fixing the issue. I have adapted the PR.

Reply:
Thanks a cool suggestion.